### PR TITLE
Hotfix 

### PR DESCRIPTION
**Pull request by victor-paumier**

### CODE REVIEW

File: src/Model/BuddyListResponse.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    public $DT_RowAttr;
+    public $section;
+
+    public function __construct(
+        TranslatorInterface $translator,
+        UrlGeneratorInterface $router,
+        \Twig_Environment $templating,
+        Buddy $buddy,
+        Section $section = null
+    )
```
Comment (JeremieSamson) [inline src/Model/BuddyListResponse.php]: ) { instead

File: src/Model/BuddyListResponse.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        $this->responseAt = ($buddy->getResponseAt()) ? datefmt_format($fmt, $buddy->getResponseAt()) : '';
+        $this->archive = ($buddy->getArchive()) ? $translator->trans('yes') : $translator->trans('no');
+        $this->actions = $templating->render('Default/listActions.html.twig', [
+            'urlShow' => $router->generate('buddy_show', ['id' => $buddy->getId()]),
+            'deleteClass' => 'buddy_delete',
+        ]);
+        $this->DT_RowAttr = ['data-id' => $buddy->getId()];
+        $this->section = $section;
+    }
+}
```
Comment (JeremieSamson) [inline src/Model/BuddyListResponse.php]: cs : line break plz

File: src/Services/MailManager.php [hunk 3]
```diff
@@ -13,12 +15,16 @@ class MailManager
...
+    private $defaultLocale;
+    private $router;
 
     public function __construct(\Twig_Environment $templating,
                                 \Swift_Mailer $mailer,
                                 TranslatorInterface $translator,
                                 LoggerHelper $logger,
+                                UrlGeneratorInterface $router,
                                 string $baseUrl,
+                                string $defaultLocale,
```
Comment (JeremieSamson) [inline src/Services/MailManager.php]: can you please fix the indentation here

Comment (victor-paumier) [reply src/Services/MailManager.php]: done

File: src/Model/BuddyListResponse.php (hunk 2 above)
Comment (victor-paumier) [reply src/Model/BuddyListResponse.php]: done

File: src/Model/BuddyListResponse.php (hunk 1 above)
Comment (victor-paumier) [reply src/Model/BuddyListResponse.php]: done

**Pull request merged**